homeassistant/components/climate/ecobee.py

```diff
@@ -253,13 +253,15 @@ def device_state_attributes(self):
         """Return device specific state attributes."""
         # Move these to Thermostat Device and make them global
         status = self.thermostat['equipmentStatus']
-        operation = None
+        aux_heat = False
```

Comment:
You changed the assignment to `operation` to `off`. This should be `None`. Also, you should set `aux_heat` to `STATE_OFF` and `STATE_ON`.

Further, I found that in `__init__.py` there is a routine `is_aux_heat_on` which you should be now implementing in ecobee, as this one is used to return the state of the aux heater in the climate component. You should then also probably rely on this property in ecobee rather than duplicating this code.